apps/confessions-client/pages/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import { Confessions } from "../components/Confessions";
+import { Login } from "../components/Login";
+import { PublishConfession } from "../components/PublishConfession";
+
+export default function Page() {
+  const [accessToken, setAccessToken] = useState<string | null>(null);
+  const [newConfession, setNewConfession] = useState<string | undefined>();
+
+  useEffect(() => {
+    if (accessToken) return;
+
+    const token = window.localStorage.getItem("access_token");
+    setAccessToken(token);
+  }, [accessToken]);
+
+  const handleAccessToken = (token: string | null) => {
```

Comment:
a better name might be `updateAccessToken`

Reply:
changed